### PR TITLE
rn-OSDOCS-4024: Add release note for SR-IOV device IDs

### PR DESCRIPTION
**Pull request by jboxman-rh**
This PR adds a release note update for supported SR-IOV hardware:

- ConnectX-6 DX

- https://issues.redhat.com/browse/OSDOCS-4024

Version(s):

-4.12
-4.11
-4.10

Issue:

Link to docs preview:

Additional information:

### CODE REVIEW

File: release_notes/ocp-4-12-release-notes.adoc [hunk 1]
```diff
@@ -74,6 +74,15 @@ For more information, see xref:../installing/installing-troubleshooting.adoc#ins
 [id="ocp-4-12-networking"]
 === Networking
 
+[id="ocp-4-12-networking-supported-hardware-for-sr-iov"]
+==== Supported hardware for SR-IOV
+
+{product-title} 4.12 adds support for the following SR-IOV devices:
```
Comment (Amrita42) [inline release_notes/ocp-4-12-release-notes.adoc:80]: Suggestion: I think SR-IOV need an expansion to _single root I/O virtualization_ for the first usage.

**Comment by Amrita42** [comment]
@jboxman-rh ptal at the conflicts . Meanwhile I 've completed the peer review.

File: release_notes/ocp-4-12-release-notes.adoc (hunk 1 above)
Comment (jboxman-rh) [reply release_notes/ocp-4-12-release-notes.adoc:80]: @Amrita42, great idea, thanks!

**Pull request merged**